lib/delegate-electron-events.js

```diff
@@ -0,0 +1,47 @@
+const EventEmitter = require('events').EventEmitter
+const electron = require('electron')
+
+const ipcMain = electron.ipcMain
+const app = electron.app
+
+module.exports = delegateElectronEvents
+
+// delegate electron events while booting
+// until a window is ready that can handle them
+// null -> EventEmitter
```

Comment:
what api comments?

I'm actually very happy with starting to use jsdoc right now so we have some self documenting code if y'all want :)